packages/affiliation-merchant-bank-accounts/src/component/MerchantBankAccounts.test.js

```diff
@@ -97,7 +94,6 @@ describe('Merchant Bank Accounts', () => {
 
   it('Should reflect "affiliationCode", "state", "isLoading", "requestErrors"' +
```

Comment:
Removeu o affiliationCode do código, mas não removeu da descrição do teste.
```suggestion
   it('Should reflect "state", "isLoading", "requestErrors"' +
```